.github/workflows/releasenotes-gen.yml

```diff
@@ -0,0 +1,19 @@
+name: "Run release note generation"
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download checkstyle
+        uses: actions/checkout@v3
+
+      - name: Generate release notes
+        env:
+          READ_ONLY_TOKEN: ${{ secrets.READ_ONLY_TOKEN_GITHUB }}
```

Comment:
Can we we use default token - GITHUB_TOKEN ?
READ_ONLY_TOKEN: ${{ secrets.GITHUB_TOKEN }}

Reply:
Done, if this token has more permissions than the RO token, I see no need to test it again.